server/chat-commands/admin.ts

```diff
@@ -1209,25 +1208,25 @@ export const commands: Chat.ChatCommands = {
 		let uhtmlId = null;
 		try {
 			/* eslint-disable no-eval, @typescript-eslint/no-unused-vars */
-			const battle = room.battle;
+			const battle = room?.battle;
 			const me = user;
 			let result = eval(target);
 			/* eslint-enable no-eval, @typescript-eslint/no-unused-vars */
 
 			if (result?.then) {
-				uhtmlId = `eval-${room.nextGameNumber()}`;
-				this.sendReply(`|uhtml|${uhtmlId}|${generateHTML('<', 'Promise pending')}`);
+				uhtmlId = `eval-${room ? room.nextGameNumber() : Date.now()}`;
```

Comment:
Honestly let's not do `nextGameNumber` at all. Just `Date.now()` + `Math.random()`.

Reply:
Actually Date.now() is millisecond precision? nvm no need for random, either

```suggestion
				uhtmlId = `eval-${Date.now()}`;
```